packages/pyolite-kernel/src/_pypi.ts

```diff
@@ -0,0 +1,8 @@
+// this file is autogenerated from the wheels described in ../package.json
+export * as allJSONUrl from '!!file-loader?name=pypi/[name].[ext]&context=.!../pypi/all.json';
+export * as ipykernelWheelUrl from '!!file-loader?name=pypi/[name].[ext]&context=.!../pypi/ipykernel-5.5.5-py3-none-any.whl';
+export * as nbformatWheelUrl from '!!file-loader?name=pypi/[name].[ext]&context=.!../pypi/nbformat-4.2.0-py3-none-any.whl';
+export * as pipliteWheelUrl from '!!file-loader?name=pypi/[name].[ext]&context=.!../pypi/piplite-0.1.0a16-py3-none-any.whl';
+export * as pyoliteWheelUrl from '!!file-loader?name=pypi/[name].[ext]&context=.!../pypi/pyolite-0.1.0a16-py3-none-any.whl';
```

Comment:
Looks like we would need to update the logic in the bump version script to handle this new location.

For now there is some basic search replace logic here:

https://github.com/jupyterlite/jupyterlite/blob/b58ea7d358cc8b7438147812739b82e1cc43abc8/scripts/bump-version.py#L44-L49